### ISSUE TITLE
Elm.embed div looses id on hot load

### ISSUE THREAD

**Issue by robinheghan** [fluxxu/elm-hot-loader#5]
Hi. I'm not sure if this is a problem with elm-hot-loader, but here it goes.

I load my app using the following snippet:

```
var Elm = require('Main.elm');
Elm.embed(Elm.Main, document.getElementById('app'), {swap:false});
```

in the index.html I have a div like this: `<div id="app"></div>`

This works great. However, when I create a change which hotloads the app. The id="app" portion of the div disappears. This has some unfortunate results when it comes to css styling, and it seems like a bug.

**Comment by fluxxu** [comment]
Hi.

This is because elm runtime will re-create container when hot swapping:

``` javascript
            function swap(newModule)
            {
                removeListeners(listeners);
                var div = document.createElement('div');
                var newElm = init(display, div, newModule, args, elm);
                inputs = [];

                return newElm;
            }
```

I can patch it, though...
Working on it now.

**Comment by fluxxu** [comment]
I just published 0.2.0.
container id and class will survive after hot swap.
Thanks.

**Comment by robinheghan** [comment]
Confirmed to be working. Thanks for your swift response :)
